pronto_mia/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -363,7 +363,7 @@
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				CURRENT_PROJECT_VERSION = 4;
```

Comment:
This was changed by Xcode when manually editing the build number. I reverted the change.
The build number has to be 4 now, because with every build we upload to App Store connect the build number has to increase.

Reply:
As long as it is the same all is well 👍🏽